distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestPlatform.java

```diff
@@ -434,24 +438,95 @@ public void testConfigurationFileWithInvalidFormat() throws IOException {
     }
 
     @Test
-    public void testExport() throws ConfigurationFileException {
-        String exportedDirectory = String.format("%s/etc/exported", ddfHome);
+    public void testExport() throws ConfigurationFileException, IOException {
         console.runCommand(EXPORT_COMMAND);
-        File[] exportedFiles = new File(exportedDirectory).listFiles();
+        File[] exportedFiles = new File(getExportDirectory()).listFiles();
         assertThat("Exported files should not be null.", exportedFiles, is(notNullValue()));
-        assertThat(String.format("No files exported to %s.", exportedDirectory),
+        assertThat(String.format("No files exported to %s.", getExportDirectory()),
```

Comment:
You should be able to use `is(not(emptyArray()))` here and not have to create a List from the array.

We should also verify that all the expected non-configuration files have been properly exported and put in the right directories, e.g., `system.properties`, `user.properties`, keystore files, etc. and that we have at least some of the expected `.config` files.